gpt_index/readers/slack.py

```diff
@@ -36,14 +37,36 @@ def __init__(self, slack_token: Optional[str] = None) -> None:
         if not res["ok"]:
             raise ValueError(f"Error initializing Slack API: {res['error']}")
 
+    def _read_message(self, channel_id: str, message_ts: str) -> str:
+        """Read a message."""
```

Comment:
would probably do import here (this allows people who haven't installed slack_sdk to still use GPT index)